src/widgets/WidgetApi.ts

```diff
@@ -34,6 +34,7 @@ export enum KnownWidgetActions {
     GetCapabilities = "capabilities",
     SendEvent = "send_event",
     UpdateVisibility = "visibility",
+    GetOpenIDCredentials = "get_openid",
```

Comment:
where is this used

Reply:
Here: https://github.com/vector-im/element-web/pull/15111/files#diff-1536b2b9d1674954ff8d3ae5d7edd952R133

Reply:
Seems like this is makinng the tests fail on the Element Web PR since it doesn't pull this version :cry: Also I noticed just that I didn't refer to `KnownWidgetActions` in all places in https://github.com/vector-im/element-web/pull/15111 .. should I refer to `KnownWidgetActions` every time or remove the reference to here (including this line of code)?

Reply:
Tests are failing because the branch names don't match

Reply:
Oh bummer. Didn't realize that was something that was needed. I'll recreate the element-web PR once fixed a bit.

Reply:
@t3chguy branch names now match - any idea why the tests are still failing to use this code here?

Reply:
looks like the branch linking broke and its trying to check out HEAD, seems like some env vars are out of shape https://buildkite.com/matrix-dot-org/element-web/builds/24#9dbc968a-9328-4951-ad69-f8f938ba80f4/60-360